### DISCUSSION TITLE
Refactor configuration

### DISCUSSION THREAD

**Discussion by dylandoamaral** [dylandoamaral/uptime-card#70] [Improvements]
I refuse to add breaking changes that force the user to change the configuration until version 1.0.0. However, version 1.0.0 should set up good foundation for the rest of the lifetime of the card.

The first thing that should be rethink and adapt is the configuration, let's discuss about it and how to make it easy to use.

Ideas for the moment (Tell me if you have some advices/opinions about configurations below):
- Replace `ok`/`ko` by `on`/`off `--> In home assistant, a binary sensor has two states `on` and `off`, we should stick with it.
- Setup sub options either by component `name`, `icon`, `timeline`, `state` or by category `color`, `show` but not both. I am not sure about which one to use but today we have both of them, maybe use only one of them.

**Comment by agneevX** [comment]
I'd recommend `on_state` and `off_state` since `on` and `off` are actual binary states.
